cleanlab_studio/utils/tlm_lite.py

```diff
@@ -110,12 +116,20 @@ def prompt(
         """
         prompt_response = self._tlm_response.prompt(prompt)
 
-        if isinstance(prompt, Sequence) and isinstance(prompt_response, list):
-            response = [r["response"] for r in prompt_response]
+        if (
+            isinstance(prompt, Sequence)
+            and isinstance(prompt_response, list)
+            and all(r["response"] is not None for r in prompt_response)
```

Comment:
I don't think there would be an instance where `r["response"] is None` here right? Since we are using `tlm.prompt()` it should just fail?

Reply:
yes, i believe i added this to pass mypy checks